CRM/Report/Form.php

```diff
@@ -3935,7 +3935,7 @@ public function addCustomDataToColumns($addFields = TRUE, $permCustomGroupIds =
           $curFilters[$fieldName]['type'] = CRM_Utils_Type::T_STRING;
 
           $options = CRM_Core_PseudoConstant::get('CRM_Core_BAO_CustomField', 'custom_' . $customDAO->cf_id, array(), 'search');
-          if ($options !== FALSE) {
+          if ((is_array($options) && count($options) != 0) || (!is_array($options) && $options !== FALSE)) {
```

Comment:
@eileenmcnaughton  I added a couple of spaces, one before "!=! and the other after "!="